music21/musicxml/test_m21ToXml.py

```diff
@@ -91,6 +91,78 @@ def testSpannersWrite(self):
         p.insert(0.0, sl3)
         self.assertEqual(self.getXml(p).count('<slur '), 6)
 
+    def testSpannerAnchors(self):
+        score = stream.Score()
+        part = stream.Part()
+        score.insert(0, part)
+        measure = stream.Measure()
+        part.insert(0, measure)
+        voice = stream.Voice()
+        measure.insert(0, voice)
+
+        n = note.Note('C', quarterLength=4)
+        sa1 = spanner.SpannerAnchor()
+        sa2 = spanner.SpannerAnchor()
+        voice.insert(0, n)
+        voice.insert(2, sa1)
+        voice.insert(4, sa2)
+        cresc = dynamics.Crescendo(n, sa1)  # cresc from n to sa1
+        dim = dynamics.Diminuendo(sa1, sa2)   # dim from sa1 to sa2
+        score.append((cresc, dim))
+
+        xmlOut = self.getXml(score)
+
+        def stripInnerSpaces(txt):
+            return re.sub(r'\s+', ' ', txt)
+
```

Comment:
I'm to blame for this. I think more standard is to use `textwrap.dedent`.

Reply:
`textwrap.dedent` doesn't do the job here (I just tried it).  Getting rid of leading spaces on each line is not the same as collapsing all whitespace to a single space.  I will get rid of the extra copy of stripInnerSpaces, though.

Reply:
Done.